app/src/main/res/layout/activity_main.xml

```diff
@@ -32,16 +32,16 @@
         android:inputType="textPersonName"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/editTextFrom" />
+        app:layout_constraintTop_toBottomOf="@id/editTextFromLocation" />
 
     <TextView
         android:id="@+id/textViewLeavingNow"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="16dp"
-        android:textSize="18sp"
         android:text="@string/textView_leaving_now"
-        app:layout_constraintBaseline_toBaselineOf="@+id/buttonChangeTime"
+        android:textSize="18sp"
```

Comment:
use material text appearances

Reply:
Replaced with:
android:textAppearance="@style/TextAppearance.AppCompat.Body1"

That's really handy